test/membrane/rtp/outbound_tracking_serializer_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule Membrane.RTP.OutboundTrackingSerializerTest do
```

Comment:
I think that we should test `RTP.Packet.serialize` instead

Reply:
I thought about it as well, but the thing is, `serialize/2` has no clue that it's creating a padding packet, it simply does the alignment to `@padding_packet_size`. For that reason, I wanted to hit `handle_process` to ensure that it detects a padding packet and calls the serialize function accordingly, especially that not working probing would be really bad to all our WebRTC products